app/components/search/SearchResults/SearchResults.jsx

```diff
@@ -102,58 +114,6 @@ const SearchResult = ({ hit, index, setCenterCoords }) => {
     return <span>No address found</span>;
   };
 
-  // Todo: to be included with next stage of multiple location work
```

Comment:
It remains to be seen if we will go back to our multiple location work or not. The codebase has already changed quite a bit since I wrote this comment, so it is probably of less relevance. And if I or anyone else wants to consult it, it will be in our history.